src/ElfUtils/ElfFile.cpp

```diff
@@ -294,25 +294,32 @@ std::filesystem::path ElfFileImpl<ElfT>::GetFilePath() const {
 template <typename ElfT>
 ErrorMessageOr<LineInfo> orbit_elf_utils::ElfFileImpl<ElfT>::GetLineInfo(uint64_t address) {
   CHECK(has_debug_info_section_);
-  auto line_info_or_error = symbolizer_.symbolizeCode(
-      *object_file_, {address, llvm::object::SectionedAddress::UndefSection});
+  auto line_info_or_error = symbolizer_.symbolizeInlinedCode(
+      object_file_->getFileName(), {address, llvm::object::SectionedAddress::UndefSection});
```

Comment:
this probably deserves a test..

Reply:
I probably have to recreate the testdata for that. Do we have a script for that?

Reply:
I honestly do not remember, I need to find it and write a readme so that I do not forget again.

Reply:
Don't worry. I will try to write a Makefile.

Reply:
So I looked into this. And it's non-trivial to recreate the hello_world_elf as there are a lot of tests which check hard-coded addresses, especially in LinuxMapTest.cpp.

So I will create an additional elf file to test the inlining stuff.

Reply:
I added a test for this. @dimitry-  PTAL

Reply:
> So I will create an additional elf file to test the inlining stuff.

Yeah, I remember now. we cannot compile because it won't produce stable result and without stable result we do not have expected values to test against.